student.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'module_high_five'
+
 class Student
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
These attributes seem similar to some of the ones in the teacher classes...  how can you refactor to account for that?  What's something that a Student and a Teacher both are?